# HG changeset patch
# User Rik <rik@octave.org>
# Date 1381114105 25200
#      Sun Oct 06 19:48:25 2013 -0700
# Node ID 4cb05034f1c6f83bc79ddce6fc8fdc7db87f89ac
# Parent  7004c733412f12b81e5f764029e613808d64f4ae
skewness.m: Use long-form of 'end' keyword in cset ab75e72c5b36.

* scripts/statistics/base/skewness.m: Replace 'end' with 'endif'.

diff --git a/scripts/statistics/base/skewness.m b/scripts/statistics/base/skewness.m
--- a/scripts/statistics/base/skewness.m
+++ b/scripts/statistics/base/skewness.m
@@ -89,17 +89,17 @@ function y = skewness (x, flag, dim)
   endif
 
   if (nargin < 2 || isempty (flag))
     flag = 1;  # default: do not use the "bias corrected" version
   else
     flag = double (flag);
     if (flag != 0 && flag != 1)
       error ("skewness: FLAG must be 0 or 1");
-    end
+    endif
   endif
 
   nd = ndims (x);
   sz = size (x);
   if (nargin < 3)
     ## Find the first non-singleton dimension.
     (dim = find (sz > 1, 1)) || (dim = 1);
   else
@@ -118,17 +118,17 @@ function y = skewness (x, flag, dim)
   y(s == 0) = NaN;
 
   ## Apply bias correction to the third central sample moment
   if (flag == 0)
     if (n > 2)
       y *= sqrt (n * (n - 1)) / (n - 2);
     else
       y(:) = NaN;
-    end
+    endif
   endif
 
 endfunction
 
 
 %!assert (skewness ([-1, 0, 1]), 0)
 %!assert (skewness ([-2, 0, 1]) < 0)
 %!assert (skewness ([-1, 0, 2]) > 0)
